contrib/annotations/README.md

```diff
@@ -0,0 +1,13 @@
+OpenTelemetry Contrib Annotations
+======================================================
+
+[![Javadocs][javadoc-image]][javadoc-url]
+
+This module contains various annotations that can be used by clients of OpenTelemetry API.
+They don't provide any functionality by themselves, but other modules, e.g. 
```

Comment:
Please clarify that they require some "kind" of byte code manipulation to work, and users who don't want to use that should not use this package?

Reply:
I think that clarification should be added to annotation's javadoc, not here. Because users will not see this file. Another option is to move this text to `package-info.java` file in `io.opentelemetry.contrib.auto.annotations` package and put a reference to it here. I even think this is a better idea than current status.

Reply:
Both are good options for me, I think using `package-info.java` may be better because we may have multiple annotations defined.

Reply:
Done.